generate_ghidra_pyi.py

```diff
@@ -50,6 +50,45 @@ def main():
     ghidra_package = type_extractor.Package.from_package(ghidra)
     type_formatter.create_type_hints(pyi_root, ghidra_package)
 
+    setup_code = """
```

Comment:
Please move the setup code generation to a different file, and just call it from here.